lib/renuo_cms_rails/cms_helper.rb

```diff
@@ -0,0 +1,28 @@
+require 'action_view/helpers'
+
+module RenuoCmsRails
+  module CmsHelper
+    def cms(path, default_value = nil, &block)
+      url_path = "#{path}-#{I18n.locale}"
```

Comment:
very good point! documented here: https://github.com/renuo/renuo-cms-rails/blob/master/README.md#cms-content-path

PS: maybe we should add this as an option (lambda)